addons/project/models/project_task.py

```diff
@@ -290,6 +290,11 @@ def _compute_analytic_account_id(self):
         for task in self:
             task.analytic_account_id = task.project_id.analytic_account_id
 
+    @api.depends('project_id')
```

Comment:
The depends is useless since the project_id field is not used inside the method. 🤔

Reply:
Changed :100:  , but I've not found yet why the runbot always fails....